### ISSUE TITLE
0.0.5 zencash stuck on "initialising" in macos sierra

### ISSUE THREAD

**Issue by redargon** [zencashio/eleos#36]
tried deleting the zen directory and restarting
tried pointing to the various z daemons
tried pointing to the zen conf file

Every startup it just sits in the initialising phase, a zen folder and conf file are made, but nothing else. 

zcl and zec open, initialise and sync perfectly

**Comment by chhudson** [comment]
+1 same issue on mac

**Comment by xdemocle** [comment]
+1 same here! stuck on "initialising" forever

**Comment by LukasBures** [comment]
1) Stop Eleos (chceck if no Eleos process run).
2) Locate your Zen folder and delete it (I dont know exact path on mac now - I will put it here later)
3) Run it, it will reinitialize again and will work

**Comment by Steamon** [comment]
I am having the same problem in windows deleted the appdata files. Redownloaded it no effects. zcl works fine.

**Comment by redargon** [comment]
When Eleos starts, there is no zend process in the process tree.If I set the wallet daemon to zcld or zcashd, then the zcld or zcashd process does start when I start Eleos. I have already tried deleting the zen folder and restarting, with all the z daemons in the Eleos folder. No joy.

**Comment by xdemocle** [comment]
yep, same here like @redargon

**Comment by Aleksandr17** [comment]
I do not work anything is infuriating! Synchronization and all not a single purse is created!

**Comment by LukasBures** [comment]
I am working on it.

**Comment by xdemocle** [comment]
@LukasBures if can helps, after following your 3 steps above and restarting eleos, I noticed that no .dat wallet has been recreated in ~/Library/Application Support/Zen. Only zen.conf

**Comment by LukasBures** [comment]
I updated eleos-macos-v0.0.5.dmg - try to download it and let me know please.

- uninstall your version
- download
- install eleos-macos-v0.0.5.dmg

**Comment by DrFireTech** [comment]
I've the same issue on windows wallet, wallet stop work if I import wallet.dat(zcl) o zen

**Comment by redargon** [comment]
Thanks for the effort so far LukasBures, but the updated dmg didn't help either. The sprout keys were fetched, no problem, the Zen folder was created with a zen.conf file, but the zend-mac daemon never starts up in the process tree when I start Eleos with ZEN as the coin. The wallet gets stuck at the initialising screen. With ZCL or ZEC as the coin, the daemons start up just fine. Setting the daemon manually also doesn't start the zend-mac daemon.

**Comment by xdemocle** [comment]
I actually didn't see any updates on releases. but confident on what @LukasBures told us, I repeated same procedures. Still same issue! No luck.

**Comment by xdemocle** [comment]
@LukasBures I tried to launch zen-cli from console.
I got the error below:
`bash-3.2$ ./zen-cli --help
dyld: Library not loaded: /usr/local/opt/gcc@5/lib/gcc/5/libgomp.1.dylib
  Referenced from: /Applications/eleos.app/Contents/Resources/app/./zen-cli
  Reason: image not found
Abort trap: 6`

**Comment by xdemocle** [comment]
and more:
`bash-3.2$ ./zen-cli --help
dyld: Symbol not found: __ZNKSt5ctypeIcE13_M_widen_initEv
  Referenced from: /Applications/eleos.app/Contents/Resources/app/./zen-cli (which was built for Mac OS X 10.12)
  Expected in: /usr/lib/libstdc++.6.dylib
 in /Applications/eleos.app/Contents/Resources/app/./zen-cli`

**Comment by Chakor** [comment]
Same problem here on El Capitan, have allowed "apps from anywhere" but is stuck on initialising. Any more tips to get it past this?
Thanks

**Comment by lucifer911** [comment]
ok so I was having all those issue that you all are having. This Video helped me.
https://www.youtube.com/watch?v=tiffSuKvAxg&t=301s
This is what I did,
Restart the computer.
Stoped any other wallet in background ( if running any ). I was running ZCL wallet.
Delete and re download the Zen oleos 0.0.5 wallet from https://github.com/zencashio/eleos/releases .
Syncing take a little time but in the end i got my zen coin. 
Hope it will help you guys.

**Comment by ecthom** [comment]
I tried everything, hope you guys can fix this soon. I got my zencash stuck in Bittrex now.

**Comment by ChurchOfNotNothing** [comment]
I've been trying to get my Zen since the blockchain has started and I didn't want to leave my ZClassic on Bittrex so I saved it in my Eleos wallet. I've been able to sucessfully get my wallet.dat file but this new Mac Eleos is stuck on initalizing after the sprout proving and verification keys get to 100%. I've tried uninstalling and reinstalling, bringing wallet.dat from the Restore ZEN from ZCL wallet.dat. Nothing works, even if I leave the name of the wallet as zcl-wallet.dat....not sure what I can do to get the 1 for 1 Zen that was promised. I'm not mad but just frustrated.

**Comment by xdemocle** [comment]
Guys/gals, @LukasBures is working on it. Please, let's be patient.

**Comment by LukasBures** [comment]
v0.0.6 try it please ... 

If that not work for you, we will need to solve signing by Apple Development ID.

**Comment by xdemocle** [comment]
![screen shot 2017-06-07 at 9 08 23 pm](https://user-images.githubusercontent.com/4141466/26896564-7fba6b10-4bc5-11e7-9467-02d6d0138056.png)
YEESSSSSS is syncing finally!!!

Thanks a lot @LukasBures !!!!

**Comment by redargon** [comment]
It works for me too. Great job @LukasBures! Thanks. Let's keep the issue open for a day or two to make sure it's working for everybody.

**Comment by logic2i** [comment]
Kick ass @LukasBures! It works and collected Zen as well. Awesome work!

**Comment by Chakor** [comment]
Champion @LukasBures. Mine is all good too, much appreciated!!

**Comment by ChurchOfNotNothing** [comment]
Patience is a virtue and @LukasBures has amazing value.

**Comment by Henock612** [comment]
Stop Eleos (chceck if no Eleos and no zend process run).
Locate your Zen folder and delete it (~/Library/Application Support/Zen)
Run it, it will reinitialize again and will work

I cant find any of these folders in with in the mac program

**Comment by dlehenky** [comment]
Just to make sure: you do realize that “~” is Unix-speak for your home directory/folder, right? The full path name for the Zen folder would be:

/Users/YOUR_USER_ID/Library/Application Support/Zen

Unfortunately, your "Library” folder doesn’t normally show in the Finder default view. In the Finder, click on the “Go” menu item. At the bottom you’ll see “Go To Folder”, select it. In the dialog box that appears, type “~/Library” and click “go”. You’ll now have a Finder window with the contents of your home “Library” folder.

HTH

-David

**Comment by Henock612** [comment]
dlehenky, Thanks for the clarification. I tried typing that in the go to folder and its not finding Application Support/Zen. I've looked through the folder and i dont see it within the v0.0.06

**Comment by dlehenky** [comment]
So, in the “Go To” dialogue box you typed “~/Library”, and the Finder window didn’t show a “Application Support” folder?

**Comment by Henock612** [comment]
yes i did see application support but not the /zen portion

**Comment by dlehenky** [comment]
You have to double-click on the “Application Support” folder to see what’s in it.

**Comment by Henock612** [comment]
yes i uploaded an image of what is in the application support folder, i don't think the zen folder is in my application support folder

**Comment by dlehenky** [comment]
Perhaps you already deleted it; maybe if/when you deleted the older version of the wallet? Just guessing that you may just be confirming that the Zen folder *has* been deleted, and you can move on to the next step. Sorry, just making my best effort to help. :)

**Comment by Henock612** [comment]
zen wallet finally sync, but now it's saying "Wallet daemon is not working on your system". how do i get the wallet daemon to work with eleos 0.0.6?
